app/locales/en.js

```diff
@@ -388,6 +388,12 @@ const en = 	{
 	witness_thread: 'witness thread',
 	you_have_votes_remaining: 'You have {votesCount} votes remaining',
 	you_can_vote_for_maximum_of_witnesses: 'You can vote for a maximum of 30 witnesses',
+	set_witness_proxy: "You can also choose a proxy that will vote for witnesses for you. This will reset your current witness selection.",
```

Comment:
please add these (as english, tagged with a `FIXME` comment) to the other language files too.